### ISSUE TITLE
Install script broken on macOS

### ISSUE THREAD

**Issue by ydalton** [vinceliuice/WhiteSur-gtk-theme#1159]
The install script makes use of a few Linuxisms that macOS does not have, and I don't know immediately how to fix them without further investigation of the code.

```
192 ~/Downloads/WhiteSur-gtk-theme-2024.09.02 % ./install.sh
readlink: illegal option -- m
usage: readlink [-fn] [file ...]
./shell/lib-core.sh: line 25: getent: command not found
```

**Comment by vinceliuice** [comment]
Anyway you cna use release files: https://github.com/vinceliuice/WhiteSur-gtk-theme/tree/master/release

**Comment by ydalton** [comment]
> Why install in macos ? it's gtk theme

I am looking into packaging my own GTK application for macOS, so I want to test if this theme will fit my needs, since the default Adwaita theme does not look great on macOS.

**Comment by vinceliuice** [comment]
MacOS and linux have different environment, so `install.sh` can not work on MacOS

**Comment by vinceliuice** [comment]
You can unzip the release file to gtk theme folder

**Comment by ydalton** [comment]
> You can unzip the release file to gtk theme folder

Yeah, so I untarred the release themes to `~/.themes`, and GTK recognizes the themes. For the record, GTK4 Widget factory looks like this on macOS:
<img width="1349" alt="Screenshot 2024-10-29 at 15 11 28" src="https://github.com/user-attachments/assets/59d96e64-c8b3-453d-9bba-45dd1285a220">
